midealocal/devices/dc/__init__.py

```diff
@@ -26,6 +36,71 @@ class DeviceAttributes(StrEnum):
 class MideaDCDevice(MideaDevice):
     """Midea DC device."""
 
+    _status: ClassVar[dict[int, str]] = {
+        1: "standby",
+        2: "start",
+        3: "pause",
+        4: "end",
+        5: "prevent_wrinkle_end",
+        6: "delay_choosing",
+        7: "fault",
+        8: "delay",
+        9: "delay_pause",
+    }
+
+    _program: ClassVar[dict[int, str]] = {
```

Comment:
Sounds similar if not identical to the  previous PR. Can we consolidate ?

Reply:
different device type have different lua protocol, we can't merge it to one.